easybuild/framework/easyconfig/types.py

```diff
@@ -116,3 +110,29 @@ def convert_value_type(val, typ):
         raise EasyBuildError("No conversion function available (yet) for target type %s", typ)
 
     return res
+
+def to_toolchain(tcspec):
+    """
+    Convert a toolchain string "foo, v0" to a dictionary {'name':'foo', 'version':'v0'}
+
+    @param tcspec: a toolchain in the form of a string or a list
+    """
+    # check if tc is a string or a list of two values; else, it can not be converted
+    if isinstance(tcspec, basestring):
+        tcspec = tcspec.split(',')
+
+    if isinstance(tcspec, list):
+        if len(tcspec) == 2:
+            return {'name':tcspec[0].strip(), 'version':tcspec[1].strip()}
+        else:
+            raise EasyBuildError("Can not convert list %s to toolchain dict. Expected 2 elements")
+    else:
+        raise EasyBuildError("Conversion of type %s to toolchain dict is not supported" % type(tcspec))
+
+TYPE_CONVERSION_FUNCTIONS = {
```

Comment:
add a comment explaining why this is at the bottom rather than the top of the module, i.e. because `to_toolchain` needs to be defined before it can be included here